test/chain-test/src/main.rs

```diff
@@ -117,9 +117,7 @@ fn main() {
         }
         Err(ref e) => {
             println!("Error: {}", e);
-            if let Some(backtrace) = e.backtrace() {
-                println!("Backtrace: {:?}", backtrace);
-            }
+            println!("Backtrace: {:?}", e.backtrace());
```

Comment:
all these test-case ones below are just b/c backtrace is baked into the `Error` type instead of being an `Option`.